### ISSUE TITLE
Allow the config file to have .yaml extension

### ISSUE THREAD

**Issue by YuriyOborozhnyi** [JasonEtco/todo#238]
## Background

Recently I run into the issue that the bot created "todo" label each PR despite the fact I deleted it and I put another default label into the `config` file.

Then I realized I had `config.yaml` under `.github` folder, but [the bot only searches for `config.yml`](https://github.com/JasonEtco/todo/blob/a15538e1d482f2579076ee092f8754c43c8d1269/lib/utils/main-loop.js#L22) (notice the slight difference in extension).

So I simply [changed extension](https://github.com/animegungrave/hugo-knowledge-base-theme/pull/31) from `.yaml` to `.yml`, and I'm fine with it, however it seems it would be good to support both of them, since `.yaml` is [official recommended extension](https://yaml.org/faq.html).

## Solution

Bot could try to find `config.yml` (it has to be done first for more backward compatibility), and if there is no such file, it could try to find `config.yaml`.

## Other Questions

- [ ] The name `config` is too generic and could lead to collisions if somebody else would like to use this name for their integration. Shouldn't it be more specific e.g. `todo-bot.yaml`?

**Comment by JasonEtco** [comment]
Hey @YuriyOborozhnyi 👋 thanks for opening this. I'm definitely a 👍 for changing it to `.github/todo.yml`. We'd still need to support `config.yml` though. If you're up for opening a PR that'd be great!

As for the actual extension - frankly, I don't think its important. I don't think we need to be making multiple requests for different extensions, it'll just slow things down. I realize that while its not "offiicial", `.yml` is very widely used.

That said, if you feel strongly about it, I'd recommend opening an issue in [probot/probot](https://github.com/probot/probot); todo consumes Probot's `context.config()` method. That's where we could talk about being more vague with both `.yaml` and `.yml`.

**Comment by YuriyOborozhnyi** [comment]
Hi @JasonEtco!

Thank you for the reply. And you're right, I don't find it important too, so np. I'll take a look at [probot/probot](https://github.com/probot/probot)'s `context.config()` method, thanks.

Would you like me to open a new issue for `.github/todo.yml` config? Or it's fine to make the change w/ just a PR?

**Comment by JasonEtco** [comment]
Just the PR is fine! Thanks ❤️
